### PR TITLE
Improving

### PR DESCRIPTION
**Pull request by Klap-in**
- usage snippets
- examples
- text improvements

### CODE REVIEW

File: about-mpdf/requirements-v7.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ permalink: /about-mpdf/requirements-v7.html
 modification_time: 2017-03-14T10:03:20+02:00
 ---
 
-mPDF 7.0 requires PHP `^5.6 || ~7.0.0 || ~7.1.0`. PHP `mbstring` (including `mbregex`, which needs to
+mPDF 7.0 requires PHP 5.6, 7.0.0 or 7.1.0. PHP `mbstring` (including `mbregex`, which needs to
```
Comment (finwe) [inline about-mpdf/requirements-v7.md]: Mayba something along `PHP >= 5.6.0 and < 7.3.0`? (7.2.* should now be compatible)

File: fonts-languages/fonts-in-mpdf-7-x.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ and `Mpdf\Language\ScriptToLanguage` implementation
 You have 2 font files `Frutiger-Normal.ttf` and `FrutigerObl-Normal.ttf` which you want to be available in mPDF,
 and you will refer to them in HTML/CSS as `frutiger`.
 
-1. Define the directory with the font via `fontDir` configuration key or add it after instatiantion
-   with `$mpdf->AddFontDirectory()` method
+1.  Define the directory with the font via `fontDir` configuration key or add it after instantiation
+    with `$mpdf->AddFontDirectory()` method
```
Comment (finwe) [inline fonts-languages/fonts-in-mpdf-7-x.md]: Why is this exactly?

Comment (Klap-in) [reply fonts-languages/fonts-in-mpdf-7-x.md]: It looks like a minor typo, i.e. http://www.dictionary.com/browse/instantiation

Comment (Klap-in) [reply fonts-languages/fonts-in-mpdf-7-x.md]: And the additional space is to improve the layouting of code blocks. It appears that without the space the code in the code blocks is prefixed with a space.

Comment (finwe) [reply fonts-languages/fonts-in-mpdf-7-x.md]: I get that, that's ok and cool, but why the doubled space? It occurs on multiple places in the PR.

Comment (Klap-in) [reply fonts-languages/fonts-in-mpdf-7-x.md]: I do not yet understand exactly what you mean by 'doubled space'?
You mean the spaces before the lines below `1.  `? these spaces are just to align the lines below the numbers with the numbers. Is supported by markdown, and looks more clear in the editor...

**Comment by finwe** [comment]
OK, cool, thanks!

**Pull request merged**